configuration/src/lib.rs

```diff
@@ -25,7 +25,7 @@ pub async fn init_tracing(service_name: &str) -> anyhow::Result<()> {
     let path_root = find_configs_root().await?;
     load_env(path_root.clone()).await?;
 
-    let mut env_filter = tracing_subscriber::EnvFilter::new(format!("{}=info", service_name));
+    let mut env_filter = tracing_subscriber::EnvFilter::new(format!("{}=info,info", service_name));
```

Comment:
I am trying to understand why it helps or doesn't work otherwise. But let it be, consider creating a housekeeping issue to handle the tracing stuff in the project someday